src/SignalR/common/Http.Connections/src/Internal/HttpConnectionsEventSource.cs

```diff
@@ -1,40 +1,50 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using System.Diagnostics.Tracing;
+using System.Threading;
 using Microsoft.Extensions.Internal;
 
 namespace Microsoft.AspNetCore.Http.Connections.Internal
 {
-    [EventSource(Name = "Microsoft-AspNetCore-Http-Connections")]
     internal class HttpConnectionsEventSource : EventSource
     {
         public static readonly HttpConnectionsEventSource Log = new HttpConnectionsEventSource();
 
-        private readonly EventCounter _connectionsStarted;
-        private readonly EventCounter _connectionsStopped;
-        private readonly EventCounter _connectionsTimedOut;
-        private readonly EventCounter _connectionDuration;
+        private PollingCounter _connectionsStartedCounter;
+        private PollingCounter _connectionsStoppedCounter;
+        private PollingCounter _connectionsTimedOutCounter;
+        private PollingCounter _currentConnectionsCounter;
+        private EventCounter _connectionDuration;
 
-        private HttpConnectionsEventSource()
+        private long _connectionsStarted;
+        private long _connectionsStopped;
+        private long _connectionsTimedOut;
+        private long _currentConnections;
+
+        internal HttpConnectionsEventSource()
+            : base("Microsoft-AspNetCore-Http-Connections")
```

Comment:
Rename to `Microsoft.AspNetCore.Http.Connections`?

Reply:
Should we also normalize the counter names? The hosting ones follow the pattern of "current-requests", and we currently use "CurrentConnections".

Reply:
Yes, lets unify the patterns.

Reply:
EventSources have always been traditionally `Microsoft-Things-Separated-By-Dashes`, why would we change that?

Also, we should update the hosting ones since these have already shipped with these names. Plus `snake-case` is so un-.NET ;)

Reply:
https://github.com/dotnet/coreclr/blob/e2b3130f4d0c9f1b759a769f4d344ee72a55b771/src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSource.cs#L12

Inconsistent AF. This is something we should resolve as part of preview7 (Hosting is updated BTW).

cc @noahfalk

Reply:
Well, it's unfortunate that CoreFX didn't follow our _completely_ undocumented design pattern from 2.2 here 😝. Hard to fix now though. I'm OK with establishing `snake-case` as the convention if that's the case.

If we do that we have two options:
* Leave the SignalR counters alone. Nobody is "broken" but they look different (though the Display Name is what is generally shown I believe)
* Change the existing SignalR counters to match the new pattern. Anybody depending on the exact name is broken. There weren't great tools for collecting these before 3.0 though so I don't feel too bad breaking that (as long as we Announcements it)

Reply:
There's only one option, use the new pattern.